src/extractors/custom/deadline.com/index.js

```diff
@@ -0,0 +1,34 @@
+export const DeadlineComExtractor = {
+  domain: 'deadline.com',
+
+  title: {
+    selectors: ['h1'],
+  },
+
+  author: {
+    selectors: ['section.author h3'],
+  },
+
+  date_published: {
+    selectors: [['meta[name="article:published_time"]', 'value']],
+  },
+
+  dek: null,
+
+  lead_image_url: {
+    selectors: [['meta[name="og:image"]', 'value']],
+  },
+
+  content: {
+    selectors: ['div.a-article-grid__main.pmc-a-grid div.pmc-a-grid-item'],
+
+    defaultCleaner: false,
```

Comment:
This one's a bit tricky! Sometimes its usage is unavoidable, but I feel like if we're trying to fix a specific issue, then we should just fix that issue instead of disabling the default cleaner, which would cause some lower-score content and elements to persist in the response content instead of being cleaned out.

It's useful to disable it on pages that heavily contain elements with very short content (such as lists, tables, etc...), but it doesn't seem to be the case here since it's content-heavy and editorial.

On this website, for example, disabling the default cleaner leads to persisting elements like:
- `.widget_pmc_outbrain_widget` and `.adma.google-publisher` (which are ad units)
- `#comments-loaded` and buttons, which you are then manually cleaning
- and some other empty/unneeded markup

What could be done, instead, is to keep the default cleaner enabled, and use `transforms` to fix the specific and known issues, which in this case I believe to be just Twitter embeds being stripped out for containing very short content. Here's the line that's responsible for this: https://github.com/postlight/mercury-parser/blob/master/src/utils/dom/clean-tags.js#L39, which is executed on the Twitter embeds because they are wrapped in `div`s. So, one way to go around this is to transform/repleace these `div`s into their inner HTML, like:

```javascript
    transforms: {
      '.embed-twitter': $node => {
        const innerHtml = $node.html();
        $node.replaceWith(innerHtml);
      },
    },
```

and then adapting the `clean` selectors as needed (you might not even need to clean anything anymore!)

Reply:
Thank you for your comment.
I added `.embed-twitter` to `transforms` as a trial.
However, two `.embed-twitter`  in the article were not included in the "content".
https://github.com/postlight/mercury-parser/blob/b7fb367958b56b1ba0d1532ea0dba6d0a9026c2f/fixtures/deadline.com/1555363730240.html#L715-L719
https://github.com/postlight/mercury-parser/blob/b7fb367958b56b1ba0d1532ea0dba6d0a9026c2f/fixtures/deadline.com/1555363730240.html#L720-L724

And one `.embed-twitter` was included without to add it to `transforms`:
https://github.com/postlight/mercury-parser/blob/b7fb367958b56b1ba0d1532ea0dba6d0a9026c2f/fixtures/deadline.com/1555363730240.html#L726-L730

Is it possible to include two `.embed-twitter` in `content`?

Reply:
Thanks for looking into this! This method and syntax for `transforms` does in fact work, as you can see in the generated ["Parsed Content Preview"](https://2771-66387122-gh.circle-artifacts.com/0/home/circleci/project/tmp/artifacts/fixtures/deadline.com/1555363730240.html.preview.html) from the ["Automated Parsing Preview"](https://github.com/postlight/mercury-parser/pull/383#issuecomment-485233569), where all 3 tweets are included in the content body.

```html
      <blockquote class="twitter-tweet">
        <p>Trump Airlines. Now there&apos;s a brand. <a
            href="https://twitter.com/realDonaldTrump?ref_src=twsrc%5Etfw">@realDonaldTrump</a> <a
            href="https://t.co/xdcgcjohW4">https://t.co/xdcgcjohW4</a></p>
        <p>&#x2014; Brian J. Karem (@BrianKarem) <a
            href="https://twitter.com/BrianKarem/status/1117767646961963009?ref_src=twsrc%5Etfw">April 15, 2019</a></p>
      </blockquote>
      <blockquote class="twitter-tweet">
        <p>Before Trump tries to act as though he knows how to fix Boeing, let&#x2019;s all review what happened to
          Trump Airlines, also known as &#x2018;Trump Shuttle&#x2019;.<a
            href="https://twitter.com/HillReporter?ref_src=twsrc%5Etfw">@hillreporter</a><a
            href="https://t.co/EFacdLsB9m">https://t.co/EFacdLsB9m</a></p>
        <p>&#x2014; Ed Krassenstein (@EdKrassen) <a
            href="https://twitter.com/EdKrassen/status/1117773751217795078?ref_src=twsrc%5Etfw">April 15, 2019</a></p>
      </blockquote>
      <p>Trump&#x2019;s tweet:</p>
      <blockquote class="twitter-tweet">
        <p>What do I know about branding, maybe nothing (but I did become President!), but if I were Boeing, I would FIX
          the Boeing 737 MAX, add some additional great features, &amp; REBRAND the plane with a new name.<br>No product
          has suffered like this one. But again, what the hell do I know?</p>
        <p>&#x2014; Donald J. Trump (@realDonaldTrump) <a
            href="https://twitter.com/realDonaldTrump/status/1117736685721223168?ref_src=twsrc%5Etfw">April 15, 2019</a>
        </p>
      </blockquote>
```
(notice that the `div.embed-twitter` is stripped out and replaced by its inner HTML, as instructed in the `transforms`)

However, the structure of the website has slightly changed, and the content selector no longer works (as mentioned in https://github.com/postlight/mercury-parser/pull/383#pullrequestreview-229403303) , which prevents the `transforms` and `clean` functions from being executed if you create a local build and try to parse a URL from this website locally; which, correct me if I'm wrong, is what I assume you're doing in order to have received incorrect results.

The reason the `transforms` worked in the PR preview (as mentioned above) is because it is using the fixture that was generated prior to the website structure change, which allowed the `content` selector to match the relevant HTML elements and run the relevant functions to transform and clean content.

Could you please regenerate this fixture and test/validate again?